src/main/java/asn1/continent.txt

```diff
@@ -0,0 +1,16 @@
+China,Asia
```

Comment:
Resource files go in `src/main/resources`. You can have subdirectories there.

Reply:
Schweet